### PR TITLE
Fix linter errors - rebase

### PR DESCRIPTION
**Pull request by cmagnobarbosa**
Fix linter errors rebase.

### CODE REVIEW

**Comment by hdiogenes** [comment]
@Exterminus could you push again so that we can see if Scrutinizer runs tests automatically now?

**Comment by cmagnobarbosa** [comment]
Fixing

File: settings.py [hunk 1]
```diff
@@ -1 +1 @@
-"""Settings for the status NApp."""
+"""Settings for the status NApp. """
```
Comment (hdiogenes) [inline settings.py]: You just have to undo this to fix linter tests again.

**Review by hdiogenes — CHANGES_REQUESTED** [review]

File: settings.py (hunk 1 above)
Comment (cmagnobarbosa) [reply settings.py]: I fixed the issue.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -23,11 +23,21 @@ Fixed
...
+[1.1.1] - 2019-03-12
+********************
+Fixed
+=====
+- Fixing linter issues and improve code organization.
+Added
+=====
+- Improve status napp.
+- Allow to run linter and tests on scrutinizer.
+
```
Comment (hdiogenes) [inline CHANGELOG.rst]: Please move changelog to the UNRELEASED section until we make a real version release.

**Review by hdiogenes — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 2 above)
Comment (cmagnobarbosa) [reply CHANGELOG.rst]: Thanks. Solved

**Pull request merged**